lectures/solow.md

```diff
@@ -387,6 +385,8 @@ linear differential equation
     x'_t = (1-\alpha) (sA - \delta x_t)
 ```
 
+To solve this equation, we need the [first ODE](https://math.libretexts.org/Bookshelves/Calculus/Calculus_(Guichard)/17%3A_Differential_Equations/17.01%3A_First_Order_Differential_Equations). 
```

Comment:
How about we delete this sentence and then change the sentence below to 

"This equation, which is a [linear ordinary differential equation](add link), has the solution"

Reply:
No problem!